src/test/resources/test_data/calculation-validation-examples.csv

```diff
@@ -25,4 +25,7 @@ validation,CRS-2186-ac1-2,,,SHPO_INVALID_DATE
 validation,CRS-2186-ac2-1,,,SHPO_INVALID_DATE
 validation,CRS-2186-ac2-2,,,SHPO_INVALID_DATE
 validation,CRS-2186-reg-1,,,
-validation,CRS-2186-reg-2,,,
\ No newline at end of file
+validation,CRS-2186-reg-2,,,
+custom-examples,crs-2162-1,,,
+custom-examples,crs-2162-2,,,
+validation,CRS-2165,,,UNABLE_TO_DETERMINE_SHPO_RELEASE_PROVISIONS
```

Comment:
Might be worth just removing this now?

Reply:
Yea I don't think there is much point in this test now.